src/main/java/JobData.java

```diff
@@ -79,7 +76,11 @@ public static ArrayList<HashMap<String, String>> findByColumnAndValue(String col
 
             String aValue = row.get(column);
 
-            if (aValue.contains(value)) {
+            // MAKE SEARCH METHODS CASE-INSENSITIVE
+            aValue = aValue.toLowerCase();
+            String bValue = value.toLowerCase();
+
+            if (aValue.contains(bValue)) {
```

Comment:
```suggestion
            if (aValue.toLowerCase().contains(value.toLowerCase())) {
```

I think this could be a shorter way to make the `findByColumnAndValue` method case insensitive.